lib/telemetry_metrics.ex

```diff
@@ -579,6 +599,20 @@ defmodule Telemetry.Metrics do
     end
   end
 
+  @spec byte_unit_conversion_ratio(byte_unit(), byte_unit()) :: number()
+  defp byte_unit_conversion_ratio(from_unit, to_unit) when from_unit == to_unit, do: 1
```

Comment:
```suggestion
  defp byte_unit_conversion_ratio(unit, unit), do: 1
```

?

Reply:
I didn't know this would work since both functions have the same arity. The compiler knows it's a different function because of the name of parameters?

Reply:
In this case it is the same arity as the functions below (arity 2). Whenever two variables in the same pattern have the same name, it means they are equal.